.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: python
 sudo: false
-python:
-    - 2.6
-    - 2.7
-    - 3.3
-    - 3.4
-    - pypy
-    - pypy3
+env:
+    - TOXENV=py26
+    - TOXENV=py27
+    - TOXENV=py32
+    - TOXENV=py33
+    - TOXENV=py34
+    - TOXENV=pypy
+    - TOXENV=pypy3
```

Comment:
I recently learned about [tox-travis](https://pypi.python.org/pypi/tox-travis).  It might make things easier/prettier.  Or not, I haven't made up my mind yet.